pkg/archive/archive_windows_test.go

```diff
@@ -36,19 +36,13 @@ func TestCopyFileWithInvalidDest(t *testing.T) {
 func TestCanonicalTarNameForPath(t *testing.T) {
 	cases := []struct {
 		in, expected string
-		shouldFail   bool
 	}{
-		{"foo", "foo", false},
-		{"foo/bar", "___", true}, // unix-styled windows path must fail
```

Comment:
Can we keep this case?

Reply:
@cpuguy83 Is is a desired behavior for these paths to fail? I would think not.

Reply:
Not fail but stay the same at least.

Reply:
Sure, added back in and updated to check they stay the same.